src/common/feedGenerator.ts

```diff
@@ -103,8 +103,8 @@ export const getExcludedAdvancedSettings = async (
 
 export const feedToFilters = async (
   con: DataSource,
-  feedId: string,
-  userId: string,
+  feedId?: string,
```

Comment:
Techically this relies on a feedId right?
Just wondering if:
A. we make it non optional but early return if not existing?
B. we keep it optional, but still early return if not existing?

Reply:
It technically was optional all the time, you can see checks inside the function to not do some queries when feedId is not defined.

Reply:
I would rather make it non optional in the future but then for anon feeds we would have to implement different functions.

Reply:
Ah I see, in the last case, let's maybe add a backlog API ticket as non urgent to fix it differently later on.